package excepciones;

public class Main
{
	public static void main(String[] args)
	{
		int[] numeros = new int[10];

		String cadena;


		try
		{
			System.out.println("Hola");
			numeros[100000] = 4;

			System.out.println("Adi√≥s");
		}
		catch(Exception e)
		{

			System.out.println("Error.");
		}
	}
}
